frontend/src/lib/lemon-ui/Popover/Popover.scss

```diff
@@ -28,20 +28,14 @@
     position: relative; // For arrow
     flex-grow: 1;
     max-width: 100%;
-    padding: 0.5rem;
```

Comment:
Padding like this that makes things that don't scroll fully to the edge are such a pet peeve of mine so thank you 😍